cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -23,6 +23,10 @@
 import java.util.Iterator;
 import java.util.LinkedHashMap;
 
+import com.google.gson.JsonElement;
+import com.google.gson.JsonNull;
+import com.google.gson.JsonPrimitive;
+import kafka.utils.Json;
```

Comment:
kafka.utils.Json never used ?

Reply:
Fixed in [690d51a](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/690d51a7a072ae9dc4b13b1f1124a9d27dbed382)